### DISCUSSION TITLE
Allow to choose which of my actions appear on for you

### DISCUSSION THREAD

**Discussion by stisa** [community/community#13404] [Other Feature Feedback, Questions, & Ideas]
I'd like to be able to choose which of my actions generate a new item on For You.
Eg if i'm reorganizing my lists, I don't want to spam other people's feed. Same if I go down a rabbit hole and start a bunch of repos.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
